File: app/src/main/java/com/example/alkewalletapp/fragmentos/LoginPage.java
package com.example.alkewalletapp.fragmentos;

import android.os.Bundle;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;
import androidx.navigation.Navigation;

import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.Button;
import android.widget.TextView;

import com.example.alkewalletapp.R;
public class LoginPage extends Fragment {


    public LoginPage() {
        // Required empty public constructor
    }

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        // Inflate the layout for this fragment
        return inflater.inflate(R.layout.fragment_login_page, container, false);
    }


    @Override
    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);

        Button botonlogin = view.findViewById(R.id.botonlogin);
        TextView crearnuevacuenta = view.findViewById(R.id.textcrearnuevacuenta);


        botonlogin.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Navigation.findNavController(v).navigate(R.id.action_loginPage_to_homePage);

            }
        });

        crearnuevacuenta.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Navigation.findNavController(v).navigate(R.id.action_loginPage_to_signupPage);
            }
        });

    }


}

File: app/src/main/java/com/example/alkewalletapp/fragmentos/LoginPage.kt
package com.example.alkewalletapp.fragmentos

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.Button
import android.widget.TextView
import androidx.fragment.app.Fragment
import androidx.navigation.Navigation.findNavController
import com.example.alkewalletapp.R

class LoginPage : Fragment() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
    }

    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        // Inflate the layout for this fragment
        return inflater.inflate(R.layout.fragment_login_page, container, false)
    }


    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)

        val botonlogin = view.findViewById<Button>(R.id.botonlogin)
        val crearnuevacuenta = view.findViewById<TextView>(R.id.textcrearnuevacuenta)


        botonlogin.setOnClickListener { v -> findNavController(v).navigate(R.id.action_loginPage_to_homePage) }

        crearnuevacuenta.setOnClickListener { v -> findNavController(v).navigate(R.id.action_loginPage_to_signupPage) }
    }
}